### PR TITLE
Week 7

### PR DESCRIPTION
**Pull request by payasbhrigu**

### CODE REVIEW

**Comment by payasbhrigu** [comment]
@anilrao316

File: Week-7/Assignment3.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const {createLinkedList, printList, Node} = require('./LinkedListCommonOperations')
```
Comment (anilrao316) [inline Week-7/Assignment3.js:1]: u can also make use of import and export statements

File: Week-7/Assignment4.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            break;
+
+        case ']':
+            top = stack.pop();
+            if (top == '(' || top == '{')
+                return false;
+            break;
+
+        case '}':
+            top = stack.pop();
```
Comment (anilrao316) [inline Week-7/Assignment4.js:29]: add default statement

**Comment by anilrao316** [comment]
rest looks good

**Comment by payasbhrigu** [comment]
Made changes as per comment.

**Pull request merged**